### DISCUSSION TITLE
Something went wrong when I try to refresh my notifications in the mobile app.

### DISCUSSION THREAD

**Discussion by Lonense** [community/community#48755] [Mobile] [answered]
Something went wrong while executing your query. Please include `16F5:2F3C:237BCF5:2526866:63FD6DA9` when reporting this issue.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.100.0 (91333308)
Device: iPhone 13 Pro (iOS 16.3.1)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi @Lonense is this issue still happening?

**Reply by Lonense** [reply]
It is fine now. Thanks.

**Reply by stevepopovich** [reply]
ok, thanks!
